orttraining/orttraining/python/training/ortmodule/_training_manager.py

```diff
@@ -177,8 +176,7 @@ def backward(ctx, *grad_outputs):
                 initializer_index = num_user_input_grads
```

Comment:
Is initializer_index` dead code after this change? Is there any performance penalty with that change? I remember some manuveurs surrounding vectors were done for performance reasons

Reply:
why it is dead code? i don't see why my change related with initializer_index.

Reply:
My bad, didn't notice line 180 still uses it